Modules/Core/EventBroker/qSlicerEventBrokerModule.cxx

```diff
@@ -53,23 +53,23 @@ vtkMRMLAbstractLogic* qSlicerEventBrokerModule::createLogic()
 //-----------------------------------------------------------------------------
 QStringList qSlicerEventBrokerModule::categories()const
 {
-  return QStringList() << "Developer Tools";
+  return QStringList() << tr("Developer Tools");
```

Comment:
This would add a “Developer Tools" translatable string under the qSlicerEventBrokerModule context. It means that this category name could be translated differently in each module. Weblate will offer to use the same string by default everywhere, but still, when changes are made then it could easily lead to inconsistencies.

What do you think about using some base class, such as `qSlicerAbstractCoreModule` for all module category name translations?

    qSlicerAbstractCoreModule::tr("Developer Tools");

Reply:
That's a good point, thank you 👍